### PR TITLE
Add get-document method to One Span module.

### PR DESCRIPTION
**Pull request by YousifHmada**
### Summary
- Implement `getDocument()` method.
- Add unit / integration tests to cover that.
- Pump pkg version to 2.1.0

### CODE REVIEW

File: test/unit/api/request.spec.js [hunk 1]
```diff
@@ -1,182 +1,294 @@
...
-const sinon = require('sinon');
-const proxyquire = require('proxyquire');
-const { Errors, Document } = require('../../../');
-const { NotImplementedError } = Errors;
+const sinon = require("sinon");
+const proxyquire = require("proxyquire");
+const { Document } = require("../../../");
 
-chai.use(require('sinon-chai'));
+chai.use(require("sinon-chai"));
```
Comment (tewen) [inline test/unit/api/request.spec.js:7]: Were double quotes in the eslint, I won't object. Just curious if that's what happened here. We may prefer single long-term.

Comment (tewen) [reply test/unit/api/request.spec.js:7]: Not a blocker. But we should change that to be single in the future.

Comment (YousifHmada) [reply test/unit/api/request.spec.js:7]: Aha, that just came up when I hit the lint-fix command

Comment (YousifHmada) [reply test/unit/api/request.spec.js:7]: > Not a blocker. But we should change that to be single in the future.

Got it 😄 👍

**Review by tewen — APPROVED** [review]

**Pull request merged**